redhat-developer/app-services-cli
# feat(service-registry rule): add base and enable command

Added service registry rule base and enable command.

### Verification Steps

1. The following command should successfully enable a registry rule:
```
./rhoas service-registry rule enable --rule-type compatibility --config forward
```
2. The following command should throw an error for invalid configuration error.
```
./rhoas service-registry rule enable --rule-type compatibility --config syntax-only
```

### Type of change


### Checklist
